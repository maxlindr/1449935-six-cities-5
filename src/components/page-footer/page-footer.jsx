import React from 'react';
import PropTypes from 'prop-types';
import {Link} from 'react-router-dom';

const PageFooter = ({container}) => (
  <footer className={container ? `footer container` : `footer`}>
    <Link className="footer__logo-link" to="/">
      <img className="footer__logo" src="img/logo.svg" alt="6 cities logo" width={64} height={33} />
    </Link>
  </footer>
);

PageFooter.propTypes = {
  container: PropTypes.bool
};

export {PageFooter};
export default PageFooter;
