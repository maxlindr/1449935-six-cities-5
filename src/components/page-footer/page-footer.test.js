import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {PageFooter} from './page-footer';

const renderer = new ShallowRenderer();

test(`PageFooter должен корректно отображаться`, () => {
  renderer.render(<PageFooter />);

  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
