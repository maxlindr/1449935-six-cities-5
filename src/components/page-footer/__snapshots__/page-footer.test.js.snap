// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter должен корректно отображаться 1`] = `
<footer
  className="footer"
>
  <Link
    className="footer__logo-link"
    to="/"
  >
    <img
      alt="6 cities logo"
      className="footer__logo"
      height={33}
      src="img/logo.svg"
      width={64}
    />
  </Link>
</footer>
`;
